components/notifications.js

```diff
@@ -484,13 +489,19 @@ function Invoicification ({ n: { invoice, sortTime } }) {
 }
 
 function WithdrawlPaid ({ n }) {
+  const payerSig = getPayerSig(n.withdrawl.invoiceForward?.[0]?.invoice?.lud18Data)
   return (
     <div className='fw-bold text-info'>
       <Check className='fill-info me-1' />{numWithUnits(n.earnedSats + n.withdrawl.satsFeePaid, { abbreviate: false, unitSingular: 'sat was ', unitPlural: 'sats were ' })}
       {n.withdrawl.p2p || n.withdrawl.autoWithdraw ? 'sent to your attached wallet' : 'withdrawn from your account'}
       <small className='text-muted ms-1 fw-normal' suppressHydrationWarning>{timeSince(new Date(n.sortTime))}</small>
       {(n.withdrawl.p2p && <Badge className={styles.badge} bg={null}>p2p</Badge>) ||
       (n.withdrawl.autoWithdraw && <Badge className={styles.badge} bg={null}>autowithdraw</Badge>)}
+      {n.withdrawl.invoiceForward?.[0]?.invoice?.comment &&
+        <small className='d-block ms-4 ps-1 mt-1 mb-1 text-muted fw-normal'>
+          <Text>{n.withdrawl.invoiceForward[0].invoice.comment}</Text>
+          {payerSig}
```

Comment:
with p2p lnurlp the notification will be a withdraw notification instead of deposit, so we implement the code to show invoice comments on withdraw too